src/test/regress/expected/pg12_0.out

```diff
@@ -0,0 +1,126 @@
+SHOW server_version \gset
```

Comment:
Well, the diff between pg12_0 and pg12_1 should be able to become empty by some normalization.
```
3c3
<  server_version_above_eleven 
---
>  server_version_above_eleven
16c16
< ERROR:  could not find function "blackhole_am_handler" in file "/home/pdube/pg11/lib/postgresql/citus.so"
---
> ERROR:  could not find function "blackhole_am_handler" in file "/home/pdube/pg10/lib/postgresql/citus.so"
85c85
< ERROR:  syntax error at or near "skip_locked"
---
> ERROR:  syntax error at or near "("
87c87
<                  ^
---
>                 ^
```
Besides, you have you local path in the error message:

> ERROR:  could not find function "blackhole_am_handler" in file "/home/pdube/pg10/lib/postgresql/citus.so"

So, we should hide this error message for sure

Reply:
Ping on this as well.